syntax = "proto3";
import "google/protobuf/empty.proto";
import "google/protobuf/timestamp.proto";

message RawMeteoDataP {
  float temperature = 1;
  float humidity = 2;
  google.protobuf.Timestamp timestamp = 3;
}

message RawPollutionDataP {
  float co2 = 1;
  google.protobuf.Timestamp timestamp = 2;
}

service DataProcessorService {
  rpc ProcessMeteoData (RawMeteoDataP) returns (google.protobuf.Empty) {}
  rpc ProcessPollutionData (RawPollutionDataP) returns (google.protobuf.Empty) {}
}

